src/InfrastructureSystems.jl

```diff
@@ -7,6 +7,7 @@ import DataFrames
 import Dates
 import JSON3
 import Random
+import Pkg
```

Comment:
We should add a note about why this is here since in theory, this is for testing.

Reply:
Not really.  It's used in the serialization process.  Is it OK to assume that a curious reader can search the codebase for "Pkg"?